Create mvp1-template11-multiomics-ctkp.json

mbrush opened the pull request:
Adding mvp1 template to support ctkp-based treats predictions  (so that we can eventually remove chembl-based templates)

mbrush committed:
Create mvp1-template11-multiomics-ctkp.json

Adding mvp1 template to support ctkp-based treats predictions  (so that we can eventually remove chembl-based templates)

karafecho reviewed (commented):
I think the `allowlist` needs to specify `infores:service-provider-trapi`. Note that there is an automat endpoint for CTKP, which would allow that service to be specifically targeted, but at the moment, there is no corresponding infores entry in the infores catalog.

mbrush commented:
Why can't we point directly to the multiomics-clinical-tirals kp in the allow list?    I think we need to point specifically to the CTKP service - as pointing to service-provider-trapi will allow predictions to be generated form chembl-based  clinical trials data that remains in the system.  

@gglusman can you advise on what infores to use here (and if a new service needs to be registered in the infores catalog, go ahead and do this?)

mbrush committed:
Update mvp1-template11-multiomics-ctkp.json

replaced trial phase based constraint with boolean elevate_to_prediction based constraint
